doc/source/tutorials/lab.rst

```diff
@@ -299,12 +342,52 @@ a two-qubit system:
                 threshold: 0.0002694329123116206
                 iq_angle: 4.912447775569025
 
+And in the case of having a chip with coupler qubits
+we need to following changes to the previous runcard:
+
+.. code-block::  yaml
+
+    qubits: [0, 1]
+    couplers: [0]
+
+    topology: {0: [0, 1]}
+
+    native_gates:
+        two_qubit:
+            0-1:
+                CZ:
+                - duration: 30
+                amplitude: 0.6025
+                shape: Rectangular()
+                qubit: 2
```

Comment:
```suggestion
                qubit: 0
```
we should change all qubits to 0, 1 because this platform is defined with two qubits only.